capa/ida/plugin/README.md

```diff
@@ -36,11 +36,15 @@ For more information on the FLARE team's open-source framework, capa, check out
 
 You can install capa explorer using the following steps:
 
-1. Install capa and its dependencies from PyPI for the Python interpreter used by your IDA installation:
+1. Install capa and its dependencies from PyPI using the Python interpreter configured for your IDA installation:
     ```
     $ pip install flare-capa
     ```
-2. Download the [standard collection of capa rules](https://github.com/mandiant/capa-rules) (capa explorer needs capa rules to analyze a database)
+2. Download and extract the [official capa ruleset](https://github.com/mandiant/capa-rules/releases) that matches the version of capa you have installed
```

Comment:
nit: i dont think "ruleset" is a word in english. we should maybe use the term "rule set".

Reply:
good catch; I'll change these before merge